### PR TITLE
Add SAP as new partner

### PR DESCRIPTION
**Pull request by jak-ing**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,7 +91,7 @@ <h1>What to expect at HackHPI 2018</h1>
 					</p>
 				</div>
 				<div class="col-md-5">
-					<img src="../hackhpi_assets/intro-bg-2018.png" class="top50 img-responsive visible-md-block visible-lg-block">
+					<img src="./hackhpi_assets/intro-bg-2018.png" class="top50 img-responsive visible-md-block visible-lg-block">
```
Comment (FWirtz) [inline index.html]: why did we change that anyways?

File: partner-information.html [hunk 2]
```diff
@@ -63,7 +63,17 @@ <h1>Partner Information</h1>
 
 				</div>
 			</div>
-			
+
+			<div class="row sponsor-information">
+				<div class="col-md-4">
+					<img src="../../hackhpi_assets/sap.png" class="img-responsive">
```
Comment (FWirtz) [inline partner-information.html]: I don't think `../../` works here, as we use `./` everywhere else. Can you confirm @jak-ing ?

File: index.html (hunk 1 above)
Comment (jak-ing) [reply index.html]: We probably copied it from the `2017/index.html` where it would be correct.

File: partner-information.html (hunk 2 above)
Comment (jak-ing) [reply partner-information.html]: Sorry, apparently I missed that one.

Comment (jak-ing) [reply partner-information.html]: Fixed @FWirtz

**Comment by FWirtz** [comment]
reverting, as I get a 404 when clicking on [Click here for further information about our partners](https://hackhpi.org/partner-information/)

File: index.html [hunk 3]
```diff
@@ -239,18 +246,16 @@ <h3 class="text-colored text-center">Supporter</h3>
...
+						<img src="./hackhpi_assets/github.png" alt="GitHub">
 					</a>
 				</div>
 			</div>
 
-			<!--
 			<div class="row">
 				<div class="col-md-12 top50">
-					<p class="text-center"><a href="partner-information/">Click here for further information about our API partners</a></p>
+					<p class="text-center"><a href="partner-information/">Click here for further information about our partners</a></p>
```
Comment (FWirtz) [inline index.html]: probably needs a `./partner-information`

Comment (FWirtz) [reply index.html]: no, just needs the bloody trailing slash removed

**Comment by FWirtz** [comment]
Fixed with abcf16e7a7ed6e7d7133557de98484452f50cf93

**Pull request merged**